stb/PSPBUILD

```diff
@@ -0,0 +1,23 @@
+pkgname=stb
+pkgver=20240714git
+pkgrel=1
+pkgdesc="a collection of single-file public domain libraries for C/C++"
+arch=('mips')
+url="https://github.com/nothings/stb"
+license=('Public Domain' 'MIT')
```

Comment:
You should use `Public Domain OR MIT` instead of splitting them into two.

ref: https://spdx.github.io/spdx-spec/v3.0/annexes/SPDX-license-expressions/ from https://wiki.archlinux.org/title/PKGBUILD#license

Reply:
Oh, I wasn't aware. Thanks! I'll change it.